src/app/modules/learning/unit/unit-item/unit-item.component.html

```diff
@@ -50,4 +50,4 @@
     <button mat-flat-button [color]="item.isNext ? 'primary' : ''" [routerLink]="['tasks', item.task.id]">Pristupi</button>
```

Comment:
We are missing the `Oceni kvalitet` button. Create it by reviewing how it works for the KC.